community/contribution_guide/become_committer.md

```diff
@@ -28,13 +28,13 @@ CoPDoC areas can become an Apache StreamPark committer. The CoPDoC is an
 acronym from ASF to describe how we recognize your contributions not
 only by code.
 
-- **Community** - You can join us via our mailing list, issue
+- **(Co)mmunity** - You can join us via our mailing list, issue
```

Comment:
Also improve the explanation of CoPDoC, it's quite confusing to me and I spend some time to understand what it means ...

Reply:
Thanks for your guidance, we refer to the descriptions of other asf projects